plugins/web/opentelemetry-instrumentation-document-load/test/documentLoad.test.ts

```diff
@@ -462,6 +462,76 @@ describe('DocumentLoad Instrumentation', () => {
       });
     });
   });
+
+  describe('add custom attributes to spans', () => {
```

Comment:
In other instrumentations, we usually also add a test to verify that a hook that throw expectation is properly cacthed and does not leak into user code